python/sdist/amici/de_model.py

```diff
@@ -1087,6 +1103,21 @@ def _generate_symbol(self, name: str) -> None:
         """
         if name in self._variable_prototype:
             components = self._variable_prototype[name]()
+            # ensure placeholder parameters are consistently and correctly ordered
+            if name == "op":
+                components = sorted(
+                    components,
+                    key=lambda x: int(
+                        x.get_name().replace("observableParameter", "")
```

Comment:
Shouldn't that be `get_id`? Why is the replacement necessary for sorting?

Reply:
We need to convert to `int` otherwise `11` will come before `2`.

Reply:
Ah, I completely missed the `int` part.